js/src/dom/event-handler.js

```diff
@@ -282,13 +281,18 @@ const EventHandler = {
     let defaultPrevented = false
     let evt = null
 
-    if (inNamespace && $) {
-      jQueryEvent = $.Event(event, args)
+    if (inNamespace) {
+      onDOMContentLoaded(() => {
```

Comment:
we don't need to wait for the DOM content to be loaded here because we don't register any jQuery plugin here